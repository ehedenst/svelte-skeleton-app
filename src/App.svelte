<script>
  import Tailwindcss from './Tailwindcss.svelte';
  export let name;
</script>

<style>
  h1 {
    @apply text-2xl;
  }
</style>

<Tailwindcss />

<h1>Hello {name}!</h1>
<div class="font-sans text-purple-500">Sans</div>
<div class="font-serif text-purple-500">Serif</div>
<div class="font-mono text-purple-500">Mono</div>
<div class="font-condensed text-purple-500">Condensed</div>
<i class="text-purple-500 material-icons md-48">face</i>
