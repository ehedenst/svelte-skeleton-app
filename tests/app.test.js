/**
 * @jest-environment jsdom
 */
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/svelte";
import App from "../src/App.svelte";

describe("App", () => {
  test("Should just work", () => expect(true).toBe(true));
  test("must render greeting", () => {
    const { getByText } = render(Comp, { props: { name: "World" } });
    expect(getByText("Hello World!")).toBeInTheDocument();
  });
  /*
  test("must render greeting", () => {
    const { getByText } = render(App, { props: { name: "world" } });

    expect(getByText("Hello world!"));
  });

  test("must snapshot", () => {
    const { container } = render(App, { props: { name: "world" } });

    expect(container).toMatchSnapshot();
  });*/
});
