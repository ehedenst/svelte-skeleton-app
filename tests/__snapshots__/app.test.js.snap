// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App must snapshot 1`] = `
<body>
  <div>
     
    <h1
      class="svelte-ber1zo"
    >
      Hello 
      world
      !
    </h1>
     
    <div
      class="font-sans text-purple-500"
    >
      Sans
    </div>
     
    <div
      class="font-serif text-purple-500"
    >
      Serif
    </div>
     
    <div
      class="font-mono text-purple-500"
    >
      Mono
    </div>
     
    <div
      class="font-condensed text-purple-500"
    >
      Condensed
    </div>
     
    <i
      class="text-purple-500 material-icons md-48"
    >
      face
    </i>
  </div>
</body>
`;
